registry/sql-registry/test/test_create.py

```diff
@@ -0,0 +1,44 @@
+import registry
+from registry.db_registry import quote
+from registry.models import AnchorDef, AnchorFeatureDef, DerivedFeatureDef, ExpressionTransformation, FeatureType, ProjectDef, SourceDef, TensorCategory, Transformation, TypedKey, ValueType, VectorType
+
+r = registry.DbRegistry()
+
+
+def cleanup():
+    with r.conn.transaction() as c:
+        ids = quote([project1_id, source1_id, anchor1_id, af1_id, df1_id])
+        c.execute(
+            f"delete from edges where from_id in ({ids}) or to_id in ({ids})")
+        c.execute(
+            f"delete from entities where entity_id in ({ids})")
+
+
+project1_id = r.create_project(ProjectDef("unit_test_project_1"))
+print("project1 id ", project1_id)
+project1 = r.get_entity(project1_id)
+assert project1.qualified_name == "unit_test_project_1"
+
+# Re-create project, should return the same id
+id = r.create_project(ProjectDef("unit_test_project_1"))
+assert project1_id == id
+
+source1_id = r.create_project_datasource(project1_id, SourceDef(
+    qualified_name="unit_test_project_1__source1", name="source1", path="hdfs://somewhere", type="hdfs"))
+print("source1 id ", source1_id)
```

Comment:
>print [](http://example.com/codeflow?start=0&length=5)

print("source1 id ", source1_id)

Should asset result instead of print result, same apply to the rest of code

Reply:
assertions added